client/state/happychat/connection/reducer.js

```diff
@@ -65,8 +66,25 @@ const isAvailable = ( state = false, action ) => {
 	return state;
 };
 
+/**
+ * Tracks whether happychat.io is accepting fully localized chats.
+ * Customers locale and HE locale should be an exact match ( doesn't default routing to english speaker HEs )
+ *
+ * @param  {Boolean} state  Current happychat status
+ * @param  {Object}  action Action playload
+ * @return {Boolean}        Updated happychat status
+ */
+const localizedSupport = ( state = false, action ) => {
```

Comment:
Do you think we should also add tests for the reducer?

Reply:
Definitely.